go/libkb/output.go

```diff
@@ -0,0 +1,18 @@
+// Copyright 2016 Keybase, Inc. All rights reserved. Use of
+// this source code is governed by the included BSD license.
+
+package libkb
+
+import (
+	"io"
+
+	"github.com/keybase/client/go/logger"
+)
+
+func OutputWriter() io.Writer {
```

Comment:
So this level of indirection was introduced [here](https://github.com/keybase/client/commit/e9b7d799).  It might actually be unrelated to logging.  It's needed to determine where the CLI outputs its error and output.  It used to be just always `os.Stdout` and `os.Stderr` before @zanderz made windows changes.